get-started/welcome.qmd

```diff
@@ -0,0 +1,108 @@
+and gives you the confidence that results are accurately rendered.
+
+```{shinylive-python}
+#| standalone: true
+#| components: [editor, viewer]
+#| layout: vertical
+#| viewerHeight: 250
+
+from shiny import reactive
+from shiny.express import ui, render, input
+
+# input slider, n
+ui.input_slider("n", "Number", min=1, max=50, value=25)
+
+# output text that uses the input, n
+@render.text
+def result():
+    return f"You selected: {input.n()}. Its value squared is {input.n()**2}."
+
+```
+
+## Templates: get started quickly {#templates}
+
+To help you hit the ground running, Shiny provides starter [templates](/templates/) for common use cases, such as data dashboards, applications, streaming updates, data entry.
+
+The `shiny create` command walks you through a series of prompts to help you get started quickly with a helpful example.
+You can learn more about the `shiny create` command and how to run your Shiny applications in the
+[create and run](create-run.qmd) section.
+
+You can use this terminal command to load up a basic dashboard:
+
+```bash
+shiny create --template dashboard
+```
+
+::::: {.column-screen-inset}
+::: {.hello-output}
+<iframe
+  src="https://gallery.shinyapps.io/template-dashboard/"
+  frameborder="0"
+  width="100%"
+  class="mb-0 card hello-output-iframe"
+></iframe>
+
+<style>
+  .hello-output-iframe {
+    height: 700px;
+  }
+@media only screen and (max-width: 611px) {
+  .hello-output-iframe {
+    height: 2125px;
+  }
+}
+</style>
+:::
+:::::
+
+
+## Extensibility: customize and expand as needed {#extensible}
+
+While Shiny includes everything you need to build an app, is built on a foundation of web standards, making it highly extensible.
+If you need a custom component or user interface, you can incrementally integrate JavaScript, HTML, or even WebAssembly without having to learn complicated build tooling and frameworks.
+
+The UI components themselves are built on a Python representation of HTML/CSS/JavaScript, which you can see by printing them in a Python REPL:
+
+```python
+>>> from shiny import ui
+>>> ui.input_action_button("btn", "Button")
+<button class="btn btn-default action-button" id="btn" type="button">Button</button>
+```
+
+If you're versed in web programming, you can also use Shiny to create [custom components](/docs/custom-component-one-off.qmd) that leverage your favorite JavaScript framework from Python.
+
+## Why choose Shiny for Python?
+
+Why Choose Shiny for Python?
+
+- 🐍 No JavaScript required – Build full-featured interactive apps in pure Python.
+- ⚡ Fast iteration – The reactive model allows quick and dynamic updates.
+- 🎨 Built-in theming and layout – Make beautiful dashboards effortlessly.
+- 🚀 Scalability and extensibility – Start simple, grow as needed.
+
+Shiny for Python empowers you to bring your data to life with interactive applications that are easy to build, customize, and share.
+
+Ready to give it a try?
```

Comment:
This kind of dangles, if I am ready to give it a try, what's my CTA?

Reply:
Added the CTA to install, and try out a template to run (links to later pages)